sklearn/linear_model/tests/test_quantile.py

```diff
@@ -252,6 +265,7 @@ def test_equivariance(quantile):
     assert_allclose(model2.coef_, np.linalg.solve(A, model1.coef_), rtol=1e-5)
 
 
+@pytest.mark.filterwarnings("ignore:`method='interior-point'` is deprecated")
```

Comment:
would using `default_solver` work here as well rather than using `interior-point` and catching the warnings?

Reply:
I cannot reproduce the failure with `"highs"` indeed.